target_chains/sui/contracts/sources/pyth.move

```diff
@@ -919,6 +926,91 @@ module pyth::pyth_tests{
         test_scenario::end(scenario);
     }
 
+    #[test]
+    // for testnet debugging
+    fun test_create_and_update_single_price_feed_with_accumulator_message_for_testnet_success() {
```

Comment:
What is the difference between this test and the previous test?If it's just the testnet, can we reuse some of the assertion logic?

Reply:
the difference is in the initial guardians, data sources, and accumulator msg. The  `expected_price_info` is different, so had to redefine that. What assertion logic do you suggest be reused?